src/InteractiveWindow/Editor/IInteractiveWindow2.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.InteractiveWindow
+{
+    /// <summary>
+    /// An implementation of a Read Eval Print Loop Window for iteratively developing code.
+    /// 
+    /// Instances of the repl window can be created by using MEF to import the IInteractiveWindowProvider interface.
+    /// </summary>
+    public interface IInteractiveWindow2 : IDisposable, IPropertyOwner
```

Comment:
Is this interface necessary? Since callers need to cast IInteractiveWindow to IInteractiveWindow2, could the callers cast IInteractiveWindow.Operations to IInteractiveWindowOperations2 instead?